ColorBox: Change demo description (#1683)
DevExpress/devextreme-demos: JSDemos/Demos/ColorBox/Overview/description.md

> The ColorBox allows users to enter a color or pick it out from the drop-down editor. When you create a ColorBox, specify its [value]() property to set an initial color. Users can change the value instantly after they picked a shade, or after they click the **OK** button. To control this behavior, specify the [applyValueMode]() property. 

Reviewer: ```suggestion
The ColorBox allows users to enter a color or select it in the drop-down editor. When you create a ColorBox, specify its [value]() property to set the initial color. Users can change the value after they pick a shade, or after they click the **OK** button. To control this behavior, specify the [applyValueMode]() property. 
```

What do you mean when you say **control**  the behavior?

Author: There are two modes: one called "instantly", when users pick a shade and its code automatically appears in the input field; the other one called "useButtons", when users pick a shade and then they need to click the OK button for code to appear in the input field. When i say "control this behavior", I mean that you can pick one of the two modes and set the applyValueMode property to "instantly" or "useButtons".

Reviewer: Perhaps:

To specify the mode, use the [applyValueMode]() property.

> You can hide the editor's drop-down button to allow users to only type in a color code. To do this, disable the [showDropDownButton]() property. Users can also edit the transparency of the picked shade if the [editAlphaChannel]() property is set to **true**.

Reviewer: ```suggestion
You can hide the editor's drop-down button to allow users to only type in a color code. To do this, disable the [showDropDownButton]() property. Users can also edit the transparency of the selected shade if the [editAlphaChannel]() property is set to **true**.
```

## Configure the ColorBox

> Use the following properties to control the component interactivity:

Reviewer: ```suggestion
Use the following properties to specify the component's interactivity:
```

Do you mean 'to specify the component's interactivity'?

Author: Yes, I mean that

- [readOnly]()    
Specifies whether the ColorBox is read-only.

- [disabled]()    
Specifies whether the ColorBox responds to user interaction.

You can specify the following properties to add custom buttons into the input field, or configure the drop-down editor's buttons:

- [buttons[]]()    
Allows you to add custom buttons to the input text field.

- [showClearButton]()    
Specifies whether to display the Clear button in the ColorBox.

- [applyButtonText]()    
Specifies the text for the button that applies changes and closes the drop-down editor.

- [cancelButtonText]()    
Specifies the text for the button that cancels changes and closes the drop-down editor.

## Handle Value Change

> Implement the [onValueChanged]() function to handle the value change when users pick a color out of the drop-down editor. To handle the input value change, use the [onInput]() function.

Reviewer: ```suggestion
Implement the [onValueChanged]() function to handle the value change when users select a color in the drop-down editor. To handle the input value change, use the [onInput]() function.
```
